packages/lesswrong/lib/generated/fragmentTypes.d.ts

```diff
@@ -890,7 +890,6 @@ interface PostsDefaultFragment { // fragment on Posts
   readonly tableOfContentsRevision: any,
   readonly sideComments: any,
   readonly sideCommentVisibility: string,
-  readonly hasFootnotes: boolean,
```

Comment:
I dunno why this change is here

Reply:
Looks like someone merged into master without running `yarn generate` (nothing major)